doc/classes/EditorExportPlugin.xml

```diff
@@ -27,6 +27,7 @@
 			<description>
 				Return [code]true[/code] if this plugin will customize scenes based on the platform and features used.
 				When enabled, [method _get_customization_configuration_hash] and [method _customize_scene] will be called and must be implemented.
+				[b]Note:[/b] [method _customize_scene] will only be called for scenes that have been modified since the last export. If no scene have been modified, it won't be called.
```

Comment:
```suggestion
				[b]Note:[/b] [method _customize_scene] will only be called for scenes that have been modified since the last export.
```

It's repeating the first sentence I think.

Reply:
I just wrote it to remove any sort of confusion left. But yeah, I think you are right 👍